pkg/webhook.v2/setup.go

```diff
@@ -18,6 +18,15 @@ package webhookv2
 
 import ctrl "sigs.k8s.io/controller-runtime"
 
-func Setup(ctrl.Manager) (string, error) {
+func Setup(mgr ctrl.Manager) (string, error) {
+	if err := setupWebhookForClusterTrainingRuntime(mgr); err != nil {
+		return "ClusterTrainingRuntime", err
+	}
+	if err := setupWebhookForTrainingRuntime(mgr); err != nil {
+		return "TrainingRuntime", err
+	}
+	if err := setupWebhookForTrainJob(mgr); err != nil {
+		return "TrainJob", err
+	}
```

Comment:
@tenzen-y Do you return these names only to handle this error: https://github.com/kubeflow/training-operator/blob/master/cmd/training-operator.v2alpha1/main.go#L150C5-L152 ?
Do you think it is necessary ? 
If some of the webhook will fail, we will get this info since we traceback error from the underlying file: ClusterTrainingRuntime, TrainingRuntime, or TrainJob.

Reply:
Yes, I believe that this name is helpful because this is not for validation error, this is for set up error.
This error can be seen when the startup webhook server failed.

Reply:
That's right, but without this name, can we see the file traceback of this error during the webhook setup ?

Reply:
I guess that the traceback will show the path where the error occurs. However, I still believe this helps understand the components in which failure occurs since this component name (ex. TrainJob) can be shown as a top-level log, and this top-level log allows us to avoid digging into nested traceback logs.

Reply:
Alright, sounds good!